private-templates-service/client/src/App.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Container } from "reactstrap";
+
+import { UserAgentApplication, ClientAuthError } from "msal";
+
+import NavBar from "./components/NavBar/NavBar";
+import ErrorMessage from "./components/ErrorMessage/ErrorMessage";
+import Welcome from "./components/Welcome/Welcome";
+import config from "./Config";
+import { getUserDetails, getOrgDetails } from "./Services/GraphService";
+import { ErrorMessageProps } from "./components/ErrorMessage/ErrorMessage";
+
+import "bootstrap/dist/css/bootstrap.css";
+
+export interface UserType {
+  displayName?: string;
+  email?: string;
+  organization?: string;
+  avatar?: string;
+}
+
+// TODO: use react-redux
+interface State {
+  isAuthenticated: boolean;
+  user: UserType;
```

Comment:
It is possible for a user to not be logged in right?  If so case we should make this able to be null or undefined. `user?: UserType`

Reply:
I think we populate user as `user: {}` and update it later on, i made that more explicit